src/app/builder/episode/episode-picker.component.ts

```diff
@@ -82,7 +82,11 @@ export class EpisodePickerComponent implements OnChanges, OnInit {
           return txt.value;
         }(item.querySelector('title').innerHTML);
 
-        let encUrl = item.querySelector('enclosure').getAttribute('url');
+        let encUrl = null;
```

Comment:
This handles when there is no enclosure in the feed, so there is no error in the parsing.